src/course-updates/update-form/UpdateForm.scss

```diff
@@ -0,0 +1,63 @@
+.update-form {
+  @include pgn-box-shadow(1, "centered");
+
+  border: .0625rem solid $gray-200;
+  border-radius: .375rem;
+  background: $white;
+  margin-bottom: 1.5rem;
```

Comment:
Try to use Paragon $spacers (`map-get($spacers, 2)`)

<img width="673" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/e35f1057-9c9e-44fd-9013-4c901080f4b8">

Reply:
Done